modules/library/src/main/scala/zio/elasticsearch/query/Queries.scala

```diff
@@ -584,6 +585,25 @@ private[elasticsearch] final case class MatchPhrase[S](field: String, value: Str
     )
 }
 
+sealed trait MultiMatchQuery[S] extends ElasticQuery[S] with HasFields[MultiMatchQuery[S]]
+
+private[elasticsearch] final case class MultiMatch[S](fields: Option[Chunk[String]], value: String)
```

Comment:
I would also add `minimum_should_match` and `boost` parameters, as they are already implemented, and can be applied to this query.

Reply:
I think it could make sense to allow something like `.matchingType(MultiMatchType.Phrase)`.
If it's more complex, it could also be a separate PR.